sdk/data_cosmos/src/clients/cosmos_client.rs

```diff
@@ -18,13 +13,10 @@ use std::sync::Arc;
 pub const EMULATOR_ACCOUNT_KEY: &str =
     "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==";
 
-const AZURE_VERSION: &str = "2018-12-31";
-
```

Comment:
Where did this go? Has this been replaced by something else?

Reply:
This is done in the `telemetry_policy` now (not a part of this PR). This can be removed since now everything goes through the pipeline and the telemetry policy will be used.